Examples/ObjectiveC/PolygonPatternExample.m

```diff
@@ -51,11 +51,11 @@ - (void)mapView:(MGLMapView *)mapView didFinishLoadingStyle:(MGLStyle *)style {
     // if your source data was added using the Mapbox Studio style editor.
     layer.sourceLayerIdentifier = @"drone-restrictions-3f6lsg";
```

Comment:
This comment needs clarification. You may want to mention that  the source layer identifier is the unique identifier of the particular layer within a vector source, and explain [how it can be obtained](https://www.mapbox.com/api-documentation/#retrieve-tilejson-metadata).